monai/utils/module.py

```diff
@@ -172,60 +168,6 @@ def damerau_levenshtein_distance(s1: str, s2: str) -> int:
     return d[string_1_length - 1, string_2_length - 1]
 
 
-def export(modname):
-    """
-    Make the decorated object a member of the named module. This will also add the object under its aliases if it has
-    a `__aliases__` member, thus this decorator should be before the `alias` decorator to pick up those names. Alias
-    names which conflict with package names or existing members will be ignored.
-    """
-
-    def _inner(obj):
-        mod = import_module(modname)
-        if not hasattr(mod, obj.__name__):
-            setattr(mod, obj.__name__, obj)
-
-            # add the aliases for `obj` to the target module
-            for alias in getattr(obj, "__aliases__", ()):
-                if not hasattr(mod, alias):
-                    setattr(mod, alias, obj)
-
-        return obj
-
-    return _inner
-
-
-def load_submodules(
```

Comment:
Do you know why we add this load_submodules here before?

Reply:
What I had implemented in my prototype ages ago was a system where you could annotate something as "export" then it would show up in the module `__init__.py` file automatically. To make this work all modules had to be loaded in two stages. This would populate members of modules but also avoid some circular import issues I'm trying to clean up right now.